plugins/modules/ec2_instance.py

```diff
@@ -808,11 +808,12 @@
 except ImportError:
     pass  # caught by AnsibleAWSModule
 
-from ansible.module_utils.six import text_type, string_types
+from ansible.module_utils.six import string_types
 from ansible.module_utils.six.moves.urllib import parse as urlparse
-from ansible.module_utils._text import to_bytes, to_native
+from ansible.module_utils._text import to_native
 import ansible_collections.amazon.aws.plugins.module_utils.ec2 as ec2_utils
 from ansible_collections.amazon.aws.plugins.module_utils.ec2 import (AWSRetry,
+                                                                     ec2_argument_spec,
```

Comment:
I assume this is just a rebase artifact, I don't think ec2_argument_spec is needed here any more, we're using AnsibleAWSModule now.

```suggestion
```